license-server.tf

```diff
@@ -28,6 +28,7 @@ resource "aws_instance" "license_server" {
                 systemctl start codemeter
                 systemctl enable codemeter
                 EOF
+  vpc_security_group_ids = [module.security_group_license_server[0].security_group_id]

```

Comment:
Is there an order in which we usually arrange the arguments of a resource?

Reply:
reordered as we discussed